gno.land/cmd/gnoland/README.md

```diff
@@ -11,3 +11,16 @@
     $> gnoland start
 
 Afterward, you can interact with [`gnokey`](../gnokey) or launch a [`gnoweb`](../gnoweb) interface.
+
+
+## Reset `gnoland` node back to genesis state. It's only suitable for testnets.
```

Comment:
Referring to https://github.com/gnolang/gno/issues/1201, to focus on a production-ready gnoland binary and maintain separate helper tools, we could simply add a contribs/gnoland-resetall, or a contribs/gnolandtools with an 'unsafe-reset-all' subcommand.